extension/background/intentParser.js

```diff
@@ -69,21 +43,36 @@ this.intentParser = (function() {
     },
     */
     play: {
-      matches: [
-        /(?:play(.*))/i,
-      ],
+      matches: [/(?:play(.*))/i],
       slots: ["query"],
     },
     pause: {
+      matches: [/^pause$/i],
+    },
+    read: {
+      matches: [/^read(?:.*)$/i, /^read(?:this )tab$/i],
+    },
+    amazonSearch: {
+      matches: [/search (?:for )?(?:a |an )?(.*) on amazon/i],
+      slots: ["query"],
+    },
+    bangSearch: {
       matches: [
-        /^pause$/i,
+        /(?:do a )?(?:search (?:my |on |for )?|query |find(?: me)? |look up |lookup |look on |look for )(google slides|google docs|spotify|goodreads|mdn|coursera|google scholar|google drive|calendar|google calendar)(?: for (?:the )?)?(.*)/i,
       ],
+      slots: ["service", "query"],
     },
-    read: {
+    bangSearchAlt: {
```

Comment:
I created #119 to remove the necessity for two intents here.

Reply:
will add a TODO w/ a reference to that issue in the code